cassandra-persistence/src/main/java/com/netflix/conductor/cassandra/dao/CassandraExecutionDAO.java

```diff
@@ -775,18 +748,9 @@ public void removeTaskFromLimit(TaskModel task) {
         try {
             recordCassandraDaoRequests(
                     "removeTaskFromLimit", task.getTaskType(), task.getWorkflowType());
-            new RetryUtil<>()
-                    .retryOnException(
-                            () ->
-                                    session.execute(
-                                            deleteTaskDefLimitStatement.bind(
-                                                    task.getTaskDefName(),
-                                                    UUID.fromString(task.getTaskId()))),
-                            null,
-                            null,
-                            3,
-                            "Deleting from task_def_limit",
-                            "removeTaskFromLimit");
+            session.execute(
```

Comment:
Although we don't use those anymore, but would it be a concern that removing the retries from `addTasktoLimit` and `removeTaskFromLimit` cause other user to start observer issues?

Reply:
I had added this more as a safeguard than an actual problem, so i think it would be ok to remove this. I do not foresee this issue to manifest and if it does it will fail the operation instead.